pom.xml

```diff
@@ -38,6 +38,8 @@
 
     <!-- the main class -->
     <exec.mainClass>org.folio.edge.orders.MainVerticle</exec.mainClass>
+    <sonar.exclusions>**/models/**.java</sonar.exclusions>
+    <sonar.test.exclusions>**/*Test.java</sonar.test.exclusions>
```

Comment:
Unless there's a really good reason for this we shouldn't be doing it.

In general you want to limit the scope of exclusions a much as possible, e.g. to a given line for a specific rule.  Even then it's good habit to add a comment explaining why it's being skipped/excluded.

Reply:
If don't skip it, then Sonar scan our test classes. As result our coverage will fall and development speed will slow down significantly. 
No make sense to check test classes for quality. And  also early test classes were skipped, but some time ago started to include in Sonar check.